viduc/casbundle/1.0/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "bundles": {
+        "Viduc\\CasBundle\\CasBundle": [
+            "all"
+        ]
+    },
+    "env": {
+        "CAS_HOST": "monserveurcas.fr",
+        "CAS_PORT": "443",
+        "CAS_VERSION": "2."
```

Comment:
This environment variable is never used.

Reply:
Thanks for your response.
I delete CAS_VERSION but not CAS_HOST and CAS_VERSION, i use it in configuration (cas.yaml)